lib/api/model/events.dart

```diff
@@ -680,24 +679,32 @@ class MessageEvent extends Event {
   // events and in the get-messages results is that `matchContent` and
   // `matchTopic` are absent here.  Already [Message.matchContent] and
   // [Message.matchTopic] are optional, so no action is needed on that.
+  @JsonKey(readValue: _readMessageValue, includeToJson: false)
   final Message message;
 
-  MessageEvent({required super.id, required this.message});
+  // The format of this is documented to be chosen freely by the client.
+  //
+  // When present, this corresponds to the JSON-encoded int, "local_id",
+  // from a previous [sendMessage] call by us.
```

Comment:
```suggestion
  // When present, this equals the "local_id" parameter
  // from a previous [sendMessage] call by us.
```

The details about what we choose to put into "local_id" are no longer something this needs in order to justify how this code works, so this can stick to documenting the API itself.